rollup-http/rollup-http-server/src/http_service.rs

```diff
@@ -106,6 +108,7 @@ async fn voucher(mut voucher: Json<Voucher>, data: Data<Mutex<Context>>) -> Http
                 e.to_string()
             );
             HttpResponse::BadRequest()
+                .append_header((hyper::header::CONTENT_TYPE, "text/plain"))
```

Comment:
Some error messages start with lower case and others with upper case. We should be consistent.